packages/TextEditor/src/TextEditor.js

```diff
@@ -1,5 +1,79 @@
 import React from "react";
+import PropTypes from "prop-types";
+import { Editor } from "@tinymce/tinymce-react";
+import { kind as kindType } from "./types";
+import * as simpleConfig from "./config/simple.config";
 
-export default function TextEditor() {
-  return <>Text editor</>;
+function getPlugins(kind) {
+  if (kind === kindType.SIMPLE) {
+    return simpleConfig.plugins;
+  }
 }
+
+function getToolbar(paramKind) {
+  if (paramKind === kindType.SIMPLE) {
+    return simpleConfig.toolbar;
+  }
+}
+
+export default function TextEditor(props) {
+  const { height, isDisabled, apiKey = "no-api-key", onChange, value, defaultValue, kind } = props;
+
+  if (value && defaultValue) {
+    throw Error("The component can only accept a value or a defaultValue prop, no both");
+  }
+
+  if (!value && !defaultValue) {
+    throw Error("The TextEditor require to have at least on prop of type value or defaultValue");
+  }
+
+  const valueFromProps = value ? { value } : { initialValue: defaultValue };
+
+  const toolbar = getToolbar(kind);
+  const plugins = getPlugins(kind);
+
+  return (
+    <Editor
+      {...valueFromProps}
+      apiKey={apiKey}
+      disabled={isDisabled}
+      init={{
+        a11y_advanced_options: true,
+        a11ychecker_html_version: "html5",
+        a11ychecker_level: "aaa",
```

Comment:
Aren't we just doing `aa`?

Reply:
good point, will setup into `aa` then...